polyply/tests/test_load_library.py

```diff
@@ -22,13 +22,17 @@
 import networkx as nx
 import vermouth
 import polyply
+from polyply import TEST_DATA
+from polyply.src.load_library import _resolve_lib_paths
+from polyply.src.load_library import read_ff_from_files
 
-#from polyply import DATA_PATH
+def test_read_ff_from_files():
+    name = "ff"
+    force_field = vermouth.forcefield.ForceField(name)
+    lib_files = _resolve_lib_paths([name], TEST_DATA)
+    read_ff_from_files(lib_files, force_field)
+
+    # Check if .ff files were parsed
+    assert force_field.blocks
+    assert force_field.links
```

Comment:
I think the test now parses the bld file but doesn't actually read it. my point being you'd need to test that the templates are actually there.